__tests__/components/TableDisplay.spec.ts

```diff
@@ -0,0 +1,1480 @@
+                runBasicShowAliasForm(
+                    'namespace',
+                    {
+                        ...mockNamespaceRowValue,
+                        aliasIdentifier: 'N/A',
+                    },
+                    {
+                        namespaceId: new NamespaceId(mockNamespaceRowValue.name),
+                        aliasTarget: null,
+                        aliasAction: 1,
+                        mosaicId: null,
+                    },
+                );
+            });
+
+            test('returns model form props when alias identifier is mosaic', () => {
+                // Arrange:
+                const alias = {
+                    aliasType: 'mosaic',
+                    aliasIdentifier: '103165893CD08625',
+                };
+
+                runBasicShowAliasForm(
+                    'namespace',
+                    {
+                        ...mockNamespaceRowValue,
+                        ...alias,
+                    },
+                    {
+                        namespaceId: new NamespaceId(mockNamespaceRowValue.name),
+                        aliasTarget: new MosaicId(alias.aliasIdentifier),
+                        aliasAction: 0,
+                        mosaicId: null,
+                    },
+                );
+            });
+
+            test('returns model form props when alias identifier is address', () => {
+                // Arrange:
+                const alias = {
+                    aliasType: 'address',
+                    aliasIdentifier: 'TCABUWAK5WMJ26ZPERMGWBOWAJF4XPNCJOWPAAI',
+                };
+
+                runBasicShowAliasForm(
+                    'namespace',
+                    {
+                        ...mockNamespaceRowValue,
+                        ...alias,
+                    },
+                    {
+                        namespaceId: new NamespaceId(mockNamespaceRowValue.name),
+                        aliasTarget: Address.createFromRawAddress(alias.aliasIdentifier),
+                        aliasAction: 0,
+                        mosaicId: null,
+                    },
+                );
+            });
+
+            test('returns model form props when alias identifier is undefined', () => {
+                // Arrange:
+                const alias = {
+                    aliasType: 'metadata',
+                    aliasIdentifier: undefined,
+                };
+
+                runBasicShowAliasForm(
+                    'namespace',
+                    {
+                        ...mockNamespaceRowValue,
+                        ...alias,
+                    },
+                    {
+                        namespaceId: new NamespaceId(mockNamespaceRowValue.name),
+                        aliasTarget: null,
+                        aliasAction: 0,
+                        mosaicId: null,
+                    },
+                );
+            });
+        });
+    });
```

Comment:
should you test form is not shown when there is no alias ?

Reply:
Actually no, it doesn't matter if alias exist or not, the form still allow to open.

https://github.com/symbol/desktop-wallet/blob/d28deaf0c6682c6c85c75446f67a16f0f73c2e71/src/components/TableDisplay/TableDisplayTs.ts#L463

If alias not exist, the form will let user to create `link` transaction.
If alias exist, the form will let user to create `unlink` transaction.